tests/test-definition.yaml

```
@@ -7,6 +7,7 @@ dimensions:
   - name: hive
     values:
       - 2.3.9-stackable0.4.0
+      - 3.1.3-stackable0.1.0
```

I think it would also make sense to test against this version as well :)
```suggestion
      - 2.3.9-stackable0.4.0
      - 3.1.3-stackable0.1.0
```